### PR TITLE
✨Add tips to the collection forms

### LINKED ISSUE #1404: 🐛 Cannot create a new Consulting page
**Describe the bug**
I tried duplicating an existing page AND creating a new page.

In both cases, when I tried to edit the SEO value I got this error:

Screenshot 2023-09-15 at 9 59 45 AM

- [x] 1. Please investigate and fix

### PR DESCRIPTION
**Pull request by KristenHu**
Add tips to the collection forms to avoid navigation bugs in Tina CMS.

- Affected routes: `/admin/index.html#/collections/new/company/~/`

- Fixed #1404 

image

### CODE REVIEW

File: .tina/collections/company.tsx [hunk 1]
```diff
@@ -20,6 +21,21 @@ export const companySchema: Collection = {
...
   fields: [
+    {
+      type: "string",
+      name: "tip",
+      label: "Tip",
+      ui: {
+        component: ({}) => {
+          return (
+            <div>
+              Please fill out all the required labels before navigating to
```
Comment (bradystroud) [inline .tina/collections/company.tsx]: Make it clear there is a field at the bottom that needs to be filled out before saving.
This is a small PR, you should be able to get this checked by anyone on the team and get it merged asap

**Review by bradystroud — CHANGES_REQUESTED** [review]

File: .tina/collections/consulting.tsx [hunk 2]
```diff
@@ -179,6 +180,21 @@ export const consultingSchema: Collection = {
...
+        component: ({}) => {
+          return (
+            <div>
+              Please fill out all the required labels before navigating to
+              another form.
+            </div>
+          );
+        },
+      },
+    },
```
Comment (Harry-Ross) [inline .tina/collections/consulting.tsx]: Can you move this to `shared-fields.tsx` so we don't violate DRY?

**Review by Harry-Ross — CHANGES_REQUESTED** [review]

**Review by Harry-Ross — APPROVED** [review]
LGTM 🚢

**Comment by jeoffreyfischer** [comment]
- **Pains**: 
1. The tip message was manually added in each page.
2. The message was not clearly saying which field to fill. 

- **Solutions**: 
1. I created a single tip message in "shared-fields" and imported it in each page.
2. The message now mentions which field needs to be filled (see screenshot)


- Affected routes: `/admin/`

- Fixed #1404 

- Screenshot

website-PR-004
**Figure**: Updated tip message.

**Review by bradystroud — APPROVED** [review]

**Pull request merged**